wrappers/swift/Swift_Wrapper-Bridging-Header.h

```diff
@@ -8,3 +8,6 @@
 #import <Foundation/Foundation.h>
 
 #import "../obj-c/ODWEventProperties.h"
+#import "../obj-c/ODWLogConfiguration.h"
+#import "../obj-c/ODWLogger.h"
```

Comment:
The two .h files in lines 12 and 13 are not part of this PR, so compile might break